src/fireblocks-sdk.ts

```diff
@@ -1159,6 +1161,13 @@ export class FireblocksSDK {
         return await this.apiClient.issueGetRequest(url);
     }
 
+    /**
+     * Get addresses for a vault account for asset with paging
+     */
+        public async getPaginatedAddresses(accountId: string, assetId: string, paginatedAddresseRequestFilters?: OptionalPaginatedAddresseRequestFilters): Promise<PaginatedAddressesResponse> {
+            return await this.apiClient.issueGetRequest(`/v1/vault/accounts/${accountId}/${assetId}/addresses_paginated?${queryString.stringify(paginatedAddresseRequestFilters)}`);
```

Comment:
queryString.stringify(...) works ok when paginatedAddresseRequestFilters is undefined ?

Reply:
yes, test it with only vaultAccountId and assetId and it works

Reply:
great